packages/terra-functional-testing/lib/config/wdio.conf.js

```diff
@@ -1,6 +1,8 @@
 const SeleniumDockerService = require('../services/wdio-selenium-docker-service');
 const TerraService = require('../services/wdio-terra-service');
 
+const { ci } = process.env;
```

Comment:
I think we will want this to be all caps

Reply:
Whoops, updated. This should be picking up the environment variable in the docker image. Might take a little trial and error on this PR.

https://github.com/cerner/terra-toolkit/pull/416/commits/736ac70c422994dc652ef8de7a52ecb2781c56fd